include/fmt/chrono.h

```diff
@@ -1319,20 +1319,20 @@ inline bool isfinite(T) {
 }
 
 // Converts value to int and checks that it's in the range [0, upper).
-template <typename T, FMT_ENABLE_IF(std::is_integral<T>::value)>
-inline int to_nonnegative_int(T value, int upper) {
+template <typename T, typename Int, FMT_ENABLE_IF(std::is_integral<T>::value)>
```

Comment:
Please update the comment in the line above because we are converting to `Int` now (with capital "I").